### DISCUSSION TITLE
Progression system V2

### DISCUSSION THREAD

**Discussion by abesto** [abesto/ams2-career#50] [Ideas]
https://github.com/abesto/ams2-career/issues/47#issuecomment-1146259086 ! :)

**Comment by abesto** [comment]
@rsr427 in https://github.com/abesto/ams2-career/issues/42#issuecomment-1146254198:
> Reaching the pinnacle unlocks a championship creator button where the season is generated at random from appropriate tracks.
Use class and car adjusted AI. Report back to the program. Get an achievement for being Champion. (Adjust AI depending on season finish?)

**Comment by abesto** [comment]
@CaptainSlow72 (I think?) / XettMan on forum.reizastudios.com, in [this comment on the Reiza forums](https://forum.reizastudios.com/threads/ams2-career-web-based-app-released.24799/page-2#post-184793):

> I don't know the structure of your database and if it can be queried for any car-/ track attributes, but what's your opinion to extend career by some additiinal series?
> 
> E.g.:
> - Asian tour: only cars and tracks from Asia
> - European tour: only cars and tracks from Europe
> - American tour: only cars and tracks from America
> - World tour: after completing the 3 series above, you can participate the world tour. Of course with content from all over the world
> 
> - Manufacturer cups: e.g. only Porsche, only Mercedes etc.
> Start from slowest car to the fastet your way up, or from the oldest to the newest
>
> Generic Forza stuff:
> - Only RWD, only FWD, only AWD, only mid engine etc.
> - Only cars < 150kW, < 250kW, >250kW, generic stuff like that.
> 
> I know, the mod was created from a spreadsheet to give motivation to try out cars and tracks from many classes and not to be a fully fledged career mode, but maybe you two guys can put it to a fully fledged career.:)

**Comment by CaptainSlow72** [comment]
What do you guys think about this:

Add some buttons to "Go race!"
"Season", "Tours" and "Career".

"Season" does what is described by rsr.
"Tours" what I described above.
"Career" let's you start as a young hopeful beginner, who climbs his way up starting with karts, amateur sunday cups, gets a contract from a professional team and ends up at formula one.
Driven by XP and/or money.
